server/db/models/Product.js

```diff
@@ -0,0 +1,71 @@
+const db = require("../db");
+const { STRING, INTEGER, DECIMAL, TEXT } = require("sequelize");
+
+///////////////// PRODUCT MODEL /////////////////
+const Product = db.define("product", {
+  name: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  // Note: the brand column will be used to pair Product with Brand via script/seed.js.
+  // john - I comment this out because this is automatically create by the association in the indexjs
+  brandName: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  imageURL: {
+    type: STRING,
+    allowNull: false,
```

Comment:
you can have a default value of like "NO IMAGE FOUND"https://www.transactis.com/wp-content/themes/unbound/images/No-Image-Found-400x264.png